### PR TITLE
Initial express api and react routes

### PR DESCRIPTION
**Pull request by jvaughn575**
This is my understanding of how express and react-router could work together. Pull it down and try it out, and then let's dicuss if this is what were after, or not.

### CODE REVIEW

**Comment by reggieroby** [comment]
add package-lock.json to the .gitignore

File: travel_hive/src/express-api/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var express = require('express');
```
Comment (reggieroby) [inline travel_hive/src/express-api/index.js]: you should move this file to the root level and rename it to server.js

File: travel_hive/src/express-api/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "express-api",
+  "version": "1.0.0",
+  "description": "express api for react frontend",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon index.js",
```
Comment (reggieroby) [inline travel_hive/src/express-api/package.json]: rename this script to "server" and add to the already created package.json.

File: travel_hive/src/express-api/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "James Vaughn",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.16.2"
```
Comment (reggieroby) [inline travel_hive/src/express-api/package.json]: express should be saved in the original package.json

File: README.md [hunk 4]
```diff
@@ -1 +1,24 @@
...
+- - - -
+
+# Steps to run: #
+
+The /src directory contains the react app and also a folder /express-api
+that holds the express api endpoint.
+
+// Nodemon will watch files in the express-script directory and restart
+// the server when files change
+1. npm install nodemon -g
```
Comment (reggieroby) [inline README.md]: i would stay away from global installs. you should install this locally and ill give you the command for it in the package.json script.

**Comment by reggieroby** [comment]
looks good, just some structure changes. you imported a whole new project into a project, and it should be incorporated into the project already created.

File: README.md (hunk 4 above)
Comment (reggieroby) [reply README.md]: something like this: "./node_modules/nodemon/bin/nodemon.js ./server.js".

this way when you "npm install" you will have all the packages needed to run our project without the need for global installs.

Comment (jvaughn575) [reply README.md]: Cool thanks for all the feedback I will make those changes and update it. 
Be hard on me, I want to get better.

Comment (reggieroby) [reply README.md]: not trying to be hard on ya, just trying to set you up for success :)

File: travel_hive/package.json [hunk 5]
```diff
@@ -3,17 +3,20 @@
...
   "dependencies": {
+    "express": "^4.16.2",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.0.17"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "start react-scripts start && start nodemon server.js",
```
Comment (reggieroby) [inline travel_hive/package.json]: really great stuff here!

Comment (reggieroby) [reply travel_hive/package.json]: this does not work. it broke my build.

**Pull request merged**